server/config.sample.json

```diff
@@ -3,7 +3,7 @@
   "RPC_PORT": 8545,
 
   "SECRET": "jpLHCCvYF0FNXkpXKfLmS7XQcOEzFhzH0f9kL1YV8YFSdIXPAArWWrRtpliILqsH",
-  "BITLY_TOKEN": <token>,
-  "FUM_TOKEN": <token>,
-  "FUM_BASEURL": <url>
+  "BITLY_TOKEN": "dummytoken",
```

Comment:
Actually I'm a bit confused, the "Some dev docs" commit is on both PRs? I guess I should look at the other one first and then this one, because now I'm going through both at the same time...

Actually neither FUM nor BITLY should be a hard dependency, there's no reason they are vital for the blockchain part, but I agree that the code is crap on this part, I can go with this if it's more clear from user perspective.